package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-lifeline",
-  "version": "6.0.2",
```

Comment:
Can we revert this? We use `release-it` to bump versions, and that process will auto-bump the version number.